fec.config.js

```diff
@@ -8,11 +8,33 @@ module.exports = {
   proxyVerbose: true,
   hotReload: true,
   plugins: [],
+  routes: {
+    ...(process.env.CONFIG_PORT && {
+      '/api/chrome-service/v1/static': {
+        host: `http://localhost:${process.env.CONFIG_PORT}`,
+      },
+      '/api/chrome-service/v1/dashboard-templates': {
+        host: `http://localhost:${process.env.CONFIG_PORT}`,
+      },
+      '/apps/frontend-starter-app': {
+        host: `http://localhost:8003`,
+      },
+      '/beta/apps/frontend-starter-app': {
+        host: `http://localhost:8003`,
+      },
+    }),
+  },
```

Comment:
This feels a bit too complicated and bound to `CONFIG_PORT`, let's use `LOCAL_WIDGET_LAYOUT` prop to load the widget layout.
```suggestion
  routes: {
    ...(process.env.CONFIG_PORT && {
      '/api/chrome-service/v1/static': {
        host: `http://localhost:${process.env.CONFIG_PORT}`,
      },
      '/api/chrome-service/v1/dashboard-templates': {
        host: `http://localhost:${process.env.CONFIG_PORT}`,
      },
    }),
    ...(process.env.LOCAL_WIDGET_LAYOUT && {
      '/apps/widget-layout': {
        host: `http://localhost:8003`,
      },
      '/beta/apps/widget-layout': {
        host: `http://localhost:8003`,
      },
    }),
```